src/python/ensembl/production/hive/ensembl_genome_metadata/MetadataUpdaterHiveProcessDb.py

```diff
@@ -22,14 +22,17 @@ class MetadataUpdaterHiveProcessDb(BaseProdRunnable):
     def fetch_input(self):
         try:
             payload = json.loads(self.param_required("payload"))
-            self.param("comment", payload["input"]["comment"])
-            self.param("database_uri", payload["input"]["database_uri"])
+
+            self.param("comment", payload["input"].get("comment", None))
+            self.param("database_uri", payload["input"].get("database_uri", None))
```

Comment:
i'd rather have those put in required params, right? We can't load metadata information without it, right?
so I believe this should be something like:
```suggestion
            self.param_required("database_uri", payload["input"].get("database_uri"))
```

Reply:
param_required doesn't work that way. It just checks. See line 28. The python code does not have the full functionality of the perl.

Reply:
I've a doubt, sorry: https://github.com/Ensembl/ensembl-hive/blob/2558e67ec603f0fe706a22143d1f953baa001d7b/wrappers/python3/eHive/process.py#L234

Reply:
Or, are you actually setting the parameter there, while retrieving it from the payload?